### ISSUE TITLE
Allow PyGrid to serve plan operations as either a list of individual commands (syft.js) or as TorchScript (Android and iOS)

### ISSUE THREAD

**Issue by cereallarceny** [OpenMined/PyGrid#437]
**Note: This issue must be done first: https://github.com/OpenMined/PySyft/issues/2994**

Once PyGrid supports the hosting of plans and protocols, and once PySyft supports the ability to compile plans as either TorchScript or as a list of procedures and stateful tensors, we must have some way for PyGrid to send the correct type of plan to the correct worker.

When a worker authenticates itself with PyGrid, it must inform PyGrid what type of plan it would like to receive: TorchScript or a list of operations. PyGrid must then retrieve the appropriate protocol and plan, and convert that plan into the appropriate format (if necessary) before returning it to the worker.

**Comment by cereallarceny** [comment]
Closing in favor of https://github.com/OpenMined/threepio/issues/49
